setup/gcp/installer/util/utils.py

```diff
@@ -298,3 +300,23 @@ def sanitize_conf_values(conf_values):
         if not conf_values[key]:
             conf_values[key] = '""'
     return conf_values
+
+def show_loading(loading_time, message='', max_number_of_dots=15):
+    """Show loading message, append dots to the end of the message up to
+    a certain number of dots and repeat.
+
+    Args:
+        loading_time (int): Loading time in seconds.
+        message (str): Message to print to stdout.
+        max_number_of_dots (int): Maximum number of dots on the line.
+    """
+
+    # VT100 control codes, use to remove the last line.
+    erase_line = '\x1b[2K'
+
+    for i in range(0, loading_time*2):
+        dots = '.' * (i % max_number_of_dots)
+        sys.stdout.write('\r{}{}{}'.format(erase_line, message, dots))
+        sys.stdout.flush()
+        time.sleep(0.5)
```

Comment:
Why multiply ```loading_time``` by 2, and then sleep for 0.5 here?  Seems simpler to leave loading_time as is, and sleep for 1 second?